### PR TITLE
Add ability to create simple stories

### PR DESCRIPTION
**Pull request by nchelluri**
@autarch what do you think of this? If you like it, I can see about adding some tests.

At MaxMind we have been using a wrapper around `App::GHPT::WorkSubmitter` to create stories and I lifted and tweaked this code from there. I fixed a bug in the process, which is why I set out to do this in the first place.

### CODE REVIEW

File: lib/App/GHPT/WorkSubmitter.pm [hunk 1]
```diff
@@ -17,6 +17,13 @@ use WebService::PivotalTracker 0.10;
...
+has create_story => (
+    is  => 'ro',
+    isa => Bool,
+    documentation =>
+        'If true, will create a new story instead of finding an existing one.',
+);
+
 has project => (
     is  => 'ro',
     isa => Str,
```
Comment (nchelluri) [inline lib/App/GHPT/WorkSubmitter.pm:29]: A coworker suggests putting a `default => $ENV{APP_GHPT_PROJECT},` in to `project`'s definition. Is that something you would be okay with?

Comment (autarch) [reply lib/App/GHPT/WorkSubmitter.pm:29]: Yes, that makes sense.

File: lib/App/GHPT/WorkSubmitter.pm [hunk 2]
```diff
@@ -105,15 +131,56 @@ has _project_ids => (
...
+    return [ map { $_->id } $projects->@* ];
+}
+
+sub _find_project ($self) {
+    my $projects = $self->_pt_api->projects;
+
+    my $want = $self->project;
+    if ($want) {
+        for my $project ( $projects->@* ) {
+            return $project if $project->name =~ /\Q$want/i;
```
Comment (2shortplanks) [inline lib/App/GHPT/WorkSubmitter.pm]: This matching isn't a good general case solution.  It won't be possible to create something reliably in project "Death Ray" if project "Death Ray v2" also exists.

File: lib/App/GHPT/WorkSubmitter.pm [hunk 3]
```diff
@@ -105,15 +131,56 @@ has _project_ids => (
...
+
+sub _find_requester ( $self, $project ) {
+    my $memberships = $project->memberships;
+
+    my $want = $self->requester;
+    if ($want) {
+        for my $membership ( $memberships->@* ) {
+            my $got_name  = fc( $membership->person->name );
+            my $want_name = fc($want);
+            return $membership->person
```
Comment (2shortplanks) [inline lib/App/GHPT/WorkSubmitter.pm]: This matching isn't a good general case solution.  It won't be possible to create something reliably for user "mark" if user "markfowler" also exists.

**Review by 2shortplanks — CHANGES_REQUESTED** [review]

File: lib/App/GHPT/WorkSubmitter.pm (hunk 2 above)
Comment (nchelluri) [reply lib/App/GHPT/WorkSubmitter.pm]: I can gather all the matches and do another `choose` on them. Thoughts?

**Review by 2shortplanks — APPROVED** [review]

**Comment by autarch** [comment]
@2shortplanks Since you merged this are you going to do the release or did you want me to do it?

**Comment by 2shortplanks** [comment]
just to confirm - I did the release

**Pull request merged**